### ISSUE TITLE
PROTOCOL: HTTP-FLV performance issue, CPU Usage too high

### ISSUE THREAD

**Issue by kiopro** [ossrs/srs#755]
Hello @winlinvip . I have one more question :)

After i push stream(RTMP) from iOS and play .flv url in webplayer or VLC,  srs usage 100% CPU why? o_o

VM: 4 core 8 gb mem 120gb ssd

<img width="648" alt="screen shot 2017-01-31 at 6 44 51 pm" src="https://cloud.githubusercontent.com/assets/5331145/22472256/25ccb1a0-e7e6-11e6-8255-697f77733280.png">

My config:
```
listen              1935;
max_connections     1000;

http_server {
    enabled         on;
    listen          8080;
    dir             ./objs/nginx/html;
}

vhost __defaultVhost__ {

    transcode {

        enabled     on;
        ffmpeg      ./objs/ffmpeg/bin/ffmpeg;

        engine 720 {
            enabled         on;
            vfilter {
            }
            vcodec          libx264;
            vbitrate        1500;
            vfps            24;
            vwidth          1278;
            vheight         720;
            vthreads        16;
            vprofile        baseline;
            vpreset         veryfast;
            vparams {
                g           100;
            }
            acodec          libfdk_aac;
            abitrate        70;
            asample_rate    44100;
            achannels       2;
            aparams {
            }
            output          rtmp://127.0.0.1:[port]/[app]?vhost=[vhost]/[stream]_[engine];
        }
    }

    ############################

    tcp_nodelay     on
    min_latency     on;

    play {
        gop_cache       off;
        queue_length    10;
        mw_latency      100;
    }

    publish {
        mr off;
    }

    ############################

    http_remux {
        enabled     on;
        mount       [vhost]/[app]/[stream].flv;
    }
}
```

**Comment by winlinvip** [comment]
What's your SRS version? How to replay this issue?

**Comment by ki0k1** [comment]
Latest version from repo (git clone), 1 stream (RTMP), 1 player (HTTP-FLV).
The load on the CPU begins at the moment when I open the HTTP-FLV link in player. If open RTMP link is ok.

**Comment by winlinvip** [comment]
Please use 2.0release branch.

**Comment by ki0k1** [comment]
@winlinvip 2.0release branch works great, thank you! 👍

**Comment by winlinvip** [comment]
Reopen for maybe bug for SRS3.

**Comment by ki0k1** [comment]
@winlinvip please check this bug in dev branch, it's very important for me. I try fix it but can't find this bug now x_x if i understand this is a problem in the muxing rtmp -> http-flv.
I also try to find the bug.~_~ I'll tell you if I find :))

**Comment by ki0k1** [comment]
hmm interesting, i can't reproduce this bug in dev branch if build srs on mac os Sierra o_0

**Comment by winlinvip** [comment]
Please DON'T use dev branch, it's not available.

**Comment by ki0k1** [comment]
@winlinvip I use 2.0release it's ok, thank you 👍  But streams in 3.0 open faster it's important for me, but .flv streams broken 😩  . And i just try fix it 😃

**Comment by winlinvip** [comment]
You can file a issue to fix the 2.0 to fast startup.

**Comment by winlinvip** [comment]
Fixed in SRS3
